lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1711,7 +1719,14 @@ def test_certificate_generation_whitelist_already_generated(self):
         }
         self.assertCertificatesGenerated(task_input, expected_results)
 
-    def test_certificate_generation_whitelisted_not_generated(self):
+    @ddt.data(
+        (CertificateStatuses.downloadable, 3),
+        (CertificateStatuses.generating, 3),
+        (CertificateStatuses.notpassing, 5),
+        (CertificateStatuses.audit_passing, 5),
```

Comment:
@saleem-latif , I'm guessing because we're not giving out audit certificates anymore. But this was how it was when I found it.